src/PackageHelper.cs

```diff
@@ -124,7 +124,7 @@ public async Task ArchivePackages(ApiKeyModel keyParts, IEnumerable<int> ids)
             {
                 using (var httpClient = GetClientBase(url, keyParts.Token, keyParts.MemberId, keyParts.ProjectId))
                 {
-                    var httpResponse = await httpClient.PostAsJsonAsync(url, new { ids });
+                    var httpResponse = await httpClient.PostAsJsonAsync(url, ids);
```

Comment:
It's not that important, but I was passing an object with IDs in just in case you ever need to extend it later as this would make that change a non-breaking change, but potentially changing the controller signature from accepting an IEnumerable to an object would be breaking.

    { id: [1,2,3,4] }

Reply:
Not sure what else we'd want to pass along, this is a pretty specific endpoint 🤔

Anyways breaking changes on Our is not as much a concern as in the CMS, and this is only used by UmbPack.

Reply:
Yea, that's cool. Probably just being overly cautious. I'm happy either way to be honest. It's not like it's something other people use. 👍